### ISSUE TITLE
How reset pm cli developer login credential in " pm auth userName userPassword"

### ISSUE THREAD

**Issue by 502530563** [PredixDev/predix-mobile-cli#3]
I didn't find any document to reset login credential or to retrieve that, if tit is there please share that.

**Comment by kamal-mann** [comment]
try using wrong credentials, ex.: `pm auth abc@xyz.com aaaaaa`
if above credentials are wrong then it will reset exisiting one.

**Comment by 502530563** [comment]
I don't think it is resetting the existing credential. It shows me error message only.

**Comment by kamal-mann** [comment]
`pm` cli tool is resetting the `auth token` here not the user credentials.
